action.yml

```
@@ -27,6 +27,17 @@ runs:
           using JuliaFormatter
           format(".") ? exit(0) : exit(1)'
       shell: bash
+    - name: Check for formatting errors
```

I think we also need something like

```suggestion
    - name: Check for formatting errors
      if: ${{ failure() }}
```

or maybe a `continue-on-error` on the previous step, or just always exit with `0` in the previous step? Because I think right now the `Format` step will fail if there is anything to format, and that would stop this step to run?

---

Yeah, good point. Removing the ternary above seems like the right way to do this.